internal/openfga/user.go

```diff
@@ -34,9 +34,19 @@ type User struct {
 	JimmAdmin bool
 }
 
-// IsAllowedAddModed returns true if the user is allowed to add a model on the
+// IsAllowedAddModelToController returns true if the user is allowed to add a model on the
```

Comment:
Can we have the setter on the user please that creates the tuple? We have this for other things.